src/python/adafruit_BNO085/jaia_firm_bno085_reset_gpio_pin.py

```diff
@@ -1,8 +1,19 @@
 import RPi.GPIO as GPIO
 import time
+from enum import Enum
+import argparse
 
-# BNO085 IMU reset pin
-bno085_pin = 10
+parser = argparse.ArgumentParser(description='Turn on/off bno085 imu', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('--imu_install_type', choices=['embedded', 'retrofit', 'none'], help='If set, configure services for imu install type')
+args=parser.parse_args()
+
+# Set BNO085 IMU reset pin
+if args.imu_install_type == 'embedded':
```

Comment:
Not sure if it's important here, but could could we remove the elif by only checking for retrofit with the if and then everything else is 10

Reply:
Yes, I will update